controlplane/hack/demo/demo-magic.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
```

Comment:
for third-party lib, it is better to reference it

Reply:
ok, removed.